dev/run-tests.py

```diff
@@ -82,6 +82,7 @@ def identify_changed_files_from_git_commits(patch_sha, target_branch=None, targe
         run_cmd(['git', 'fetch', 'origin', str(target_branch+':'+target_branch)])
     else:
         diff_target = target_ref
+    run_cmd(['git', 'rebase', diff_target])
```

Comment:
Thank you for review, @JoshRosen .
What side-effects do you think specifically?
I think that's normal process to test old PR to the master.